add justfile

pwinckles opened the pull request:
This PR adds a convenience [Just](https://github.com/casey/just) integration. The only purpose is to make development easier by removing the need to look up complicated commands. Some examples:

Running the tests in docker:

```shell
just test-build-image
just test
```

Building a fresh image and running FITS on a local file:

```shell
just build-image
just run -i my-cool-file.pdf
```

awoods reviewed (changes requested):
When I pull down this branch and run you example command (`just build-image`), not surprisingly, I get a "Command 'just' not found" error. I would be helpful to add a note about `just` and its usage (and optionally its installation) in the [README.md](https://github.com/harvard-lts/fits#readme).

pwinckles commented:
> When I pull down this branch and run your example command (`just build-image`), not surprisingly, I get a "Command 'just' not found" error. It would be helpful to add a note about `just` and its usage (and optionally its installation) in the [README.md](https://github.com/harvard-lts/fits#readme).

Will do. I'll update it after the formatting pr is merged

pwinckles commented:
> When I pull down this branch and run your example command (`just build-image`), not surprisingly, I get a "Command 'just' not found" error. It would be helpful to add a note about `just` and its usage (and optionally its installation) in the [README.md](https://github.com/harvard-lts/fits#readme).

I rebased the PR and added notes to the readme

awoods reviewed (approved):
Works as advertised... nice.
